### ISSUE TITLE
Set md-Minlength

### ISSUE THREAD

**Issue by adriendomoison** [formly-js/angular-formly-templates-material#6]
Hello,

I'm trying to understand how works the ngModelAttrs when setting-up the input type, I see that it's possible to set properties like that : 

    defaultOptions: {
        ngModelAttrs: {
            mdMaxlength: {
                bound: 'md-maxlength'
            }
        }
    }

but I don't really understand where does the 'md-maxlength' come from and when I tried to do the same with minlength like that : 

    defaultOptions: {
        ngModelAttrs: {
            mdMaxlength: {
                bound: 'md-maxlength'
            },
            mdMinlength: {
                bound: 'md-minlength'
            }
        }
    }

it doesn't work and I really don't understand why and how it works ^^
Could you explain me please? :)

Thank you !

**Comment by kentcdodds** [comment]
Yeah, that api is pretty complicated. Have you looked at the docs?

- Kent C. Dodds

(Sent from my mobile device, please forgive typos or brevity)
